src/test/java/org/apache/sysds/test/functions/federated/algorithms/FederatedLmPipeline.java

```diff
@@ -63,7 +63,6 @@ public void federatedLmPipelineSampled() {
 		federatedLmPipeline(Types.ExecMode.SINGLE_NODE, false, TEST_NAME1);
 	}
 
-	@Test
```

Comment:
should this be added again?

if it is failing and you don't intend to fix it add an "@ Ignore" instead